packages/vite/src/modes/shadow-dom.ts

```diff
@@ -4,8 +4,8 @@ import { IMPORT_CSS } from '../../../plugins-common'
 
 export function ShadowDomModuleModePlugin({ uno }: UnocssPluginContext): Plugin {
   const partRegex = /^part-\[.*]:/
+  const partExtractorRegex = /^part-\[(.+)]:/
```

Comment:
I think the bracket capture should be the same.

https://github.com/antfu/unocss/blob/7c08eab2f016d3867f7820ea3250acfb11223b82/packages/preset-mini/src/variants/pseudo.ts#L58